### DISCUSSION TITLE
How can I run a Windows program from CLI with Bottles Flatpak?

### DISCUSSION THREAD

**Discussion by felipehw** [bottlesdevs/Bottles#137] [Q&A] [answered]
Hi, I'm new to Bottles.

How can I run a Windows program from CLI with Bottles Flatpak?

**Comment by mirkobrombin** [comment]
I never tried with the Flatpak but try:

```
flatpak run com.usebottles.bottles executable_path.exe
```

**Reply by felipehw** [reply]
It didn't work.
I created an issue at Flathub repo: <https://github.com/flathub/com.usebottles.bottles/issues/14>

**Comment by mirkobrombin** [comment]
I will investigate

**Reply by felipehw** [reply]
It will be awesome if we can do something like: `bottles  -b merlot myapp.exe`. So, we can easily create our own `.desktop` files.
I'm using Silverblue (an immutable linux that depends, mainly, on containers and flatpaks). Actually, I'm using Wine through a container. But if Bottles Flatpak works via CLI ... I would replace my container for Bottles. So my `.desktop` files will have something like `flatpak run --command=bottles com.usebottles.bottles -b merlot myapp.exe`. :D

**Reply by mirkobrombin** [reply]
> It will be awesome if we can do something like: `bottles  -b merlot myapp.exe`. So, we can easily create our own `.desktop` files.
> I'm using Silverblue (an immutable linux that depends, mainly, on containers and flatpaks). Actually, I'm using Wine through a container. But if Bottles Flatpak works via CLI ... I would replace my container for Bottles. So my `.desktop` files will have something like `flatpak run --command=bottles com.usebottles.bottles -b merlot myapp.exe`. :D
> 

There a project to bring a cli on the next bottles release (v4) https://github.com/bottlesdevs/Bottles/issues/133

Bottles v3 can run .exe from cli only with AppImage or native packages

**Reply by felipehw** [reply]
I'll wait for v4. Thanks.

**Reply by mirkobrombin** [reply]
> I'll wait for v4. Thanks.

There is a lot of work to be done for v4 but I confirm that this feature will be there

**Comment by mirkobrombin** [comment, marked as answer]
Bottles CLI will be available wth Bottles v4.
